index.html

```diff
@@ -259,6 +262,50 @@ <h1 class="about-section-h1">About Me</h1>
           </ul>
         </div>
       </section>
+      <section class="form-section">
+        <div class="formtext">
+          <p>I'am always<br> interested in hearing about new projects, so<br> if you'd like to <br> please get in</p>
+        </div>
+        <div class="main-form">
+          <form action="#">
```

Comment:
- [ ] action value should be similar to https://formspree.io/f/{form_id}. Please kindly fix that.
- [ ] `method` value equals `POST`. Please kindly fix that.